src/chigame/games/views.py

```diff
@@ -19,6 +22,28 @@ class LobbyListView(SingleTableView):
     template_name = "games/lobby_list.html"
 
 
+@login_required
+def lobby_join(request, pk):
+    lobby = get_object_or_404(Lobby, pk=pk)
+    joined = Lobby.objects.filter(members=request.user.id)
+    print(joined, lobby)
+    if lobby in joined:
+        return HttpResponseNotFound("Already joined.")
+    lobby.members.add(request.user)
+    return redirect(reverse("lobby-details", kwargs={"pk": lobby.id}))
+
+
+@login_required
+def lobby_leave(request, pk):
+    lobby = get_object_or_404(Lobby, pk=pk)
+    joined = Lobby.objects.filter(members=request.user.id)
+    print(joined, lobby)
+    if lobby not in joined:
+        return HttpResponseNotFound("Haven't Joined.")
+    lobby.members.remove(request.user)
+    return redirect(reverse("lobby-details", kwargs={"pk": lobby.id}))
+
+
 class ViewLobbyDetails(DetailView):
     model = Lobby
     template_name = "games/lobby_details.html"
```

Comment:
Tested the code using these commands:
- `python3 manage.py testserver src/chigame/games/games-data.json src/chigame/users/fixtures/users_fixture.json`
- Logged in as `user1@example.com`:`test`

Everything works as intended. I think next steps might include:
- only offering the "join lobby" button to users who have not yet joined the lobby (rather than offering it and redirecting to an error page)
- similarly, only offering the "leave lobby" button to users who have joined the lobby
- updating the Members list to accurately reflect the joined users (right now, it says "None" even though the user has joined)
But I recognize that that is not the purpose of this PR. Well done!

Approved with the caveat that I think you should add a /fixtures folder and move the fixture into it, which will allow us to better keep track of all our future fixtures. :)